packages/lab/stories/link-card/link-card.stories.css

```diff
@@ -0,0 +1,40 @@
+.card-demo-inner {
```

Comment:
How much of this is needed for the link card stories? This currently mentions interactable card

Reply:
potentially none